### DISCUSSION TITLE
ComboBox options prop

### DISCUSSION THREAD

**Discussion by jlee-enquizit** [trussworks/react-uswds#2541] [Q&A]
Using the ComboBox component, typing into the input fires an onChange that makes API call to return a new array for the options dropdown.
The props are updated but the menu does not
What could be going on here?

**Comment by werdnanoslen** [comment]
Hi @jlee-enquizit, would you be able to share more details or if https://github.com/trussworks/react-uswds/pull/2715 's story gives some perspective
